demisto_sdk/commands/common/hook_validations/description.py

```diff
@@ -147,28 +149,43 @@ def verify_demisto_in_description_content(self):
         Return:
             True if 'Demisto' does not exist in the description content, and False if it does.
         """
-
-        data_dictionary = get_yaml(self.file_path)
-        is_unified_integration = data_dictionary.get('script', {}).get('script', '') not in {'-', ''}
+        description_path = ''
         yml_line_num = 0
+        yml_or_file = ''
 
-        if is_unified_integration:
-            description_content = data_dictionary.get('detaileddescription', '')
+        # case 1 the file path is for an integration
+        if find_type(self.file_path) == FileType.INTEGRATION:
```

Comment:
what about beta integrations? use the ignore sub-categories

Reply:
right